cmd/getbalance.go

```diff
@@ -29,6 +32,15 @@ var getbalancecmd = &cobra.Command{
 		}
 		b := statusBar.balance
 		usd, err := zcncore.ConvertTokenToUSD(b.ToToken())
+		
+		if doJSON {
+			j := make(map[string]string)
+			j["usd"] = fmt.Sprintf("%f", usd)
+			j["zcn"] = fmt.Sprintf("%f", b.ToToken())
+			j["fmt"] = fmt.Sprintf("%s", b)
+			util.PrintJSON(j)
```

Comment:
We can it in this way:
```go
j := map[string]string {
  "usd": fmt.Sprintf("%f", usd),
  "zcn": fmt.Sprintf("%f", b.ToToken())
...
}
```

Reply:
Sure I see this is more compact, please implement as you see fit